GVFS/GVFS.Tests/NUnitRunner.cs

```diff
@@ -42,6 +42,12 @@ public int RunTests(ICollection<string> includeCategories, ICollection<string> e
                 this.args.Add(filters);
             }
 
+            // If there are any test filters, the GlobalSetup still needs to run so add it.
+            if (this.args.Any(x => x.StartsWith("--test=")))

```

Comment:
Looks like unit tests are broken when using `--test=` because of
```
    [SetUpFixture]
    public class Setup
    {
        [OneTimeSetUp]
        public void SetUp()
        {
            GVFSPlatform.Register(new MockPlatform());
        }
    }
```

not running for the same reason.  Let me see what I can do to fix both.